src/engraving/libmscore/textbase.cpp

```diff
@@ -2783,7 +2783,7 @@ bool TextBase::validateText(QString& s)
                 d.append("&amp;");
             }
         } else if (c == '<') {
-            const char* ok[] { "b>", "/b>", "i>", "/i>", "u>", "/u", "font ", "/font>", "sym>", "/sym>" };
+            const char* ok[] { "b>", "/b>", "i>", "/i>", "u>", "/u", "font ", "/font>", "sym>", "/sym>", "sub>", "/sub>", "sup>", "/sup>" };
             QString t = s.mid(i + 1);
```

Comment:
Obviously this change will affect quite a lot more than just instrument names, but I don't see any harm in that.